src/main/java/com/configcat/ConfigFetcher.java

```diff
@@ -152,7 +152,7 @@ private CompletableFuture<FetchResponse> getResponseAsync(String eTag) {
         this.httpClient.newCall(request).enqueue(new Callback() {
             @Override
             public void onFailure(@NotNull Call call, @NotNull IOException e) {
-                String generalMessage = "An error occurred during fetching the latest config.json.";
+                String generalMessage = "An error occurred during fetching of the latest config.json.";
```

Comment:
If this means the same, then it can be the same as above:
`Exception while fetching the config.json` or `Exception while trying to fetch the config.json`